src/test/java/io/odpf/depot/message/proto/ProtoOdpfParsedMessageTest.java

```diff
@@ -325,6 +328,20 @@ public void shouldGetFieldByName() throws IOException {
         Assert.assertEquals("order-1", protoOdpfParsedMessage.getFieldByName("order_number", odpfMessageSchema));
     }
 
```

Comment:
can we add more tests? non repeated but complex, nested, timestamp?

Reply:
added tests for timestamp, duration, map. Repeated and nested fields already exist.